assignments/this.js

```diff
@@ -1,26 +1,69 @@
 /* The for principles of "this";
 * in your own words. explain the four principle for the "this" keyword below.
 *
-* 1. 
-* 2. 
-* 3. 
-* 4. 
+* 1. Global - this will call the window of the browser because the binding of this is Global
+* 2. Implicit - this will call the object to the left of the "." where the function is called
```

Comment:
Slightly better wording is that `this` would be whatever the object is left of the dot. What is being "called" is the method (_a function which is a property of an object_).